cluster/gce/gci/configure-helper.sh

```diff
@@ -1506,12 +1506,6 @@ function set_docker_options_non_ubuntu() {
 
    addockeropt "\"mtu\": 1460,"
```

Comment:
these should also be moved a level up then? Since the config file is overrriden, it doesn't matter what were the default any longer

Reply:
live-restore and storage-driver is already as an input in the ubuntu preloading image. Adding these to daemon.json might result in failure.